suite-native/app/app.config.ts

```diff
@@ -84,6 +84,10 @@ const getPlugins = (): ExpoPlugins => {
             {
                 backgroundColor: '#25292E',
                 image: './assets/splash_icon.png',
+                dark: {
```

Comment:
The same color and image? 🤔

Reply:
Might be sufficient to change the color. Who knows...